docs/05-device-support.md

```diff
@@ -1,17 +1,18 @@
 
-# balenaSound device type support
+# Supported devices
 
+balenaSound has been tested to work on the following devices:
 
-| Device Type  | Multi-room disabled (`DISABLE_MULTI_ROOM=1`) | Multi-room server | Multi-room client |
+| Device Type  | Multi-room disabled | Multi-room server | Multi-room client |
 | ------------- | ------------- | ------------- | ------------- |
-| Raspberry Pi (v1 / Zero / Zero W) | ✔ | ✘ [1] | ✔ |
-| Raspberry Pi 2 | ✔ | [2] | [2] | 
-| Raspberry Pi 3 [3] | ✔ | ✔ [4] | ✔ | 
-| Raspberry Pi 4 [3] | ✔ | ✔ | ✔ | 
+| Raspberry Pi (v1 / Zero / Zero W) | ✔ | ✘ ([1]) | ✔ |
+| Raspberry Pi 2 | ✔ | ✘ ([1]) | [2] | 
+| Raspberry Pi 3 ([3]) | ✔ | ✔ ([4]) | ✔ | 
+| Raspberry Pi 4 ([3]) | ✔ | ✔ | ✔ | 
 
```

Comment:
Add balenaFin? I think this could help people, which are planning to use balenaSound for a fleet of balenaFin's.

Reply:
@tmigone do you think we could use the 3D designs of boombeastic? https://github.com/balenalabs/boombeastic/tree/master/designs

I know that the project is old, but i like the design of Curcuz. Making balenaSound portable, would be very cool. Do you agree?

Reply:
Unfortunately the fin has no audio jack 😭 but with a DAC, why not! I'll see if I can spin it up and test it in the next couple days!

About boombeastic, yeah Carlo still working in balena, the designs should be there and work. Maybe @chrisys can put his 3d printer to use 😀 ?

Reply:
@tmigone i just realised that with the audio jack 😅. David sent me two balenaFins 1.0. Shall i test multiroom? 

Chris has a 3D printer? Cool! The 3D models contain the resin.io logo. That should be updated. 

I think some changes should be made for portable functionality. I thought of my GPIO Bluetooth control and GPIO Pi wakeup. Cool features for a DIY speaker.